data/json/items/gun/357.json

```diff
@@ -14,7 +14,7 @@
     "symbol": "(",
     "color": "dark_gray",
     "ammo": "357sig",
-    "dispersion": 400,
+    "dispersion": 480,
     "durability": 7,
```

Comment:
That's not recoil, that's dispersion. Dispersion affects all shots.

Reply:
Mistake in the commit message. I meant dispersion.